oak-search-elastic/src/test/java/org/apache/jackrabbit/oak/plugins/index/elastic/ElasticIndexProviderServiceTest.java

```diff
@@ -95,6 +96,8 @@ public void defaultSetup() {
 
     @Test
     public void withElasticSetup() throws Exception {
+        // Skips this test when a single Elasticsearch cluster is re-used among test executionsgit , see the ElasticTestServer main.
+        assumeTrue(elasticRule.useDocker());
```

Comment:
That's not correct: the tests need to work also with single ES clusters. Please revert these changes. To fix the problem lines 105 and 124 need to be replaced with (I have tried to push the commit myself but I have problem s with this branch):
```java
props.put(PROP_ELASTIC_PORT, elasticRule.getElasticConnectionModel().getElasticPort());
```

Reply:
The assumeTrue(elasticRule.useDocker()) statement should also be removed from both the tests in which it was added. Otherwise they will only run on a docker based setup.

Reply:
I removed it